gtfs/migrations/0012_add_feed_info.py

```diff
@@ -0,0 +1,88 @@
+# Generated by Django 3.1.7 on 2021-04-12 20:02
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("gtfs", "0011_add_missing_fields"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="FeedInfo",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "publisher_name",
+                    models.CharField(max_length=255, verbose_name="publisher name"),
+                ),
+                ("publisher_url", models.URLField(verbose_name="publisher URL")),
+                (
+                    "lang",
+                    models.CharField(
+                        help_text="Default language for the text in this dataset, used by the feed importer. IETF BCP 47 language code",
+                        max_length=16,
+                        verbose_name="language",
+                    ),
+                ),
+                (
+                    "default_lang",
+                    models.CharField(
+                        blank=True,
+                        help_text="Defines the language used when the data consumer doesn’t know the language of the rider. IETF BCP 47 language code",
+                        max_length=16,
+                        verbose_name="default language",
+                    ),
+                ),
+                (
+                    "start_date",
+                    models.DateField(blank=True, null=True, verbose_name="start date"),
+                ),
+                (
+                    "end_date",
+                    models.DateField(blank=True, null=True, verbose_name="end date"),
+                ),
+                (
+                    "version",
+                    models.CharField(blank=True, max_length=64, verbose_name="version"),
+                ),
+                (
+                    "contact_email",
+                    models.EmailField(
+                        blank=True, max_length=254, verbose_name="contact email"
+                    ),
+                ),
+            ],
+            options={
+                "verbose_name": "feed info",
+                "verbose_name_plural": "feed info",
+            },
+        ),
+        migrations.AddConstraint(
```

Comment:
Just noticed that this constraint was removed in migration `0011`, but good that it is re-added here. :+1:

Reply:
Good catch. Strange, I wonder why it was removed in `0011`